_test/test_sqw_pageOpMethods/test_sqw_copy.m

```diff
@@ -49,7 +53,7 @@ function test_copy_excluding_pix_returns_empty_pix_data(obj)
             % confirm selected other data is copied
             assertEqual(sqw_copy.main_header.title, sqw_obj.main_header.title);
             assertEqualToTol(sqw_copy.experiment_info, sqw_obj.experiment_info);
-            assertEqualToTol(sqw_copy.detpar(), sqw_obj.detpar());
+            %assertEqualToTol(sqw_copy.detpar(), sqw_obj.detpar());
         end
```

Comment:
such changes make me feel like something fishy going on. Why it has been commented? is it incorrect?

Reply:
detpar has been removed as a separate datum, hence the commenting out. the test still works with the () function call parentheses removed, as the detpar dependent property channels the detector arrays. Formally this is a duplication of the test on experiment_info in the previous line. It is retained here without the () as a test of the detpar dependent variable usage.